superset-frontend/src/components/Chart/DrillBy/DrillByMenuItems.tsx

```diff
@@ -253,20 +323,23 @@ export const DrillByMenuItems = ({
           ) : filteredColumns.length ? (
             <div
               css={css`
-                max-height: ${MAX_SUBMENU_HEIGHT}px;
-                overflow: auto;
+                height: ${MAX_SUBMENU_HEIGHT}px;
               `}
             >
-              {filteredColumns.map(column => (
-                <MenuItemWithTruncation
-                  key={`drill-by-item-${column.column_name}`}
-                  tooltipText={column.verbose_name || column.column_name}
-                  {...rest}
-                  onClick={e => handleSelection(e, column)}
-                >
-                  {column.verbose_name || column.column_name}
-                </MenuItemWithTruncation>
-              ))}
+              <AutoSizer>
+                {({ height, width }: { height: number; width: number }) => (
+                  <List
+                    width={width}
+                    height={height}
```

Comment:
Since the height is already fixed, what do you think about locking the height value into the same variable to improve resize performance?
Additionally I would also like to avoid using the autosizer if we can set the width to a specific size, since there are numerous callback functions linked to resize events within the charts on the current dashboard.

```suggestion
                {({ width }: { width: number }) => (
                  <List
                    width={width}
                    height={MAX_SUBMENU_HEIGHT}
```

Reply:
That makes total sense. Great suggestion.